ste/sender-azureFile.go

```diff
@@ -34,14 +34,27 @@ import (
 	"github.com/Azure/azure-storage-azcopy/common"
 )
 
+type URLHolder interface {
+	URL() url.URL
+	String() string
+}
+
+// TODO: review the decision to have this implement both interfaces
```

Comment:
makes sense

Reply:
Just checking, @zezha-msft . Did you mean "makes sense" to leave it as it is?  Or "make sense" to review and reconsider the current approach?

Reply:
Sorry, makes sense to leave as it is